source/common/listener_manager/lds_api.cc

```diff
@@ -77,7 +77,9 @@ LdsApiImpl::onConfigUpdate(const std::vector<Config::DecodedResourceRef>& added_
       failure_state.push_back(std::make_unique<envoy::admin::v3::UpdateFailureState>());
       auto& state = failure_state.back();
       state->set_details(error_message);
+#if defined(ENVOY_ENABLE_FULL_PROTOS)
```

Comment:
Do we know where "mutable_failed_configurations" is used, and what will happen if it's empty in this case? Is it just logged? Oh, looks like just used in a test. Great!